packages/web/src/components/card/desktop/Card.module.css

```diff
@@ -1,28 +1,9 @@
-.cardContainer {
-  cursor: pointer;
+.root {
   position: relative;
-  border: 1px solid var(--neutral-light-8);
-  border-radius: 8px;
   display: inline-flex;
   flex-direction: column;
   align-items: center;
-  background-color: var(--white);
-  box-shadow: 0 0 1px 0 var(--tile-shadow-1), 0 1px 0 0 var(--tile-shadow-2),
-    0 2px 5px -2px var(--tile-shadow-3);
-  transition: all 0.18s ease-in-out;
-}
-
-.cardContainer:hover {
-  background-color: var(--neutral-light-10);
-  box-shadow: 0 1px 5px 1px var(--tile-shadow-1-alt),
-    0 1px 0 0 var(--tile-shadow-2), 0 2px 10px -2px var(--tile-shadow-3);
-  transform: scale3d(1.01, 1.01, 1.01);
-}
```

Comment:
Confirming that we don't need the hover states anymore?

Reply:
yeah they all come from the tile now